packages/vscode-extension/src/webviews/connectionPanel.ts

```diff
@@ -45,6 +46,7 @@ export class ConnectionPanel {
     this._extensionPath = extensionPath;
     this._connection = connection;
     this._password = password;
+    this._editConnection = connection ? true : false;
```

Comment:
Now that we retain information about the connection when we change to another tab,  we need this to distinguish between a connection that is being edited vs one that we are adding.